remark/mermaid/style.css

```diff
@@ -87,12 +91,11 @@ g.label foreignObject {
   overflow-x: visible;
 }
 
-.edgeLabel div {
-  background-color: var(--mermaid-er-label-fill);
-}
-
 .edgeLabel span.edgeLabel {
   background-color: transparent;
+
+  -webkit-text-stroke-color: #fff;
+  -webkit-text-stroke-width: 4px;
```

Comment:
Browser support seems fine and note:

> [Firefox & Edge specifically only support the `-webkit-text-stroke` property (not using `-moz-` or `-ms-` prefix)](https://caniuse.com/?search=-webkit-text-stroke)

Unfortunately this doesn't seem to work on SVG `text` so there's some dodgy `stroke`ing there instead.